Detectors/TOF/calibration/src/TOFFEElightReader.cxx

```diff
@@ -22,20 +22,21 @@ void TOFFEElightReader::loadFEElightConfig(const char* fileName)
   char* expandedFileName = gSystem->ExpandPathName(fileName);
   std::ifstream is;
   is.open(expandedFileName, std::ios::binary);
-  is.read(reinterpret_cast<char*>(&mFEElightConfig), sizeof(mFEElightConfig));
+  char* tempbuf = new char[sizeof(o2::tof::TOFFEElightConfig)];
+  is.read(tempbuf, sizeof(o2::tof::TOFFEElightConfig));
   is.close();
+  mFEElightConfig = reinterpret_cast<TOFFEElightConfig*>(tempbuf);
```

Comment:
the tempbuf should be a class data member, which is reset on reading a new file and cleaned on destructing the object, e.g.
````
std::unique_ptr<char[]> mFileLoadBuff; // in the class declaration
...
mFileLoadBuff.reset(  new char[sizeof(o2::tof::TOFFEElightConfig)] );
is.read(mFileLoadBuff.get(), sizeof(o2::tof::TOFFEElightConfig));
is.close();
mFEElightConfig = reinterpret_cast<TOFFEElightConfig*>(mFileLoadBuff.get());
````

Reply:
Done. 
Would it there be a leak in the previous version?  
And, for my knowledge, you suggested to use a `unique_ptr` because in this way I don't have to manually delete anything, right when a new file comes, right? The default destructor would take care of deleting a standard pointer, if I am not wrong.

Reply:
Yes, the previous version would lead to a leak on every reading and also because the destructor does not clean the the memory pointed by the pointer data member, it just destroys the pointer itself (which in your case was a local variable, anyway)